sdk/handleError.js

```diff
@@ -10,30 +10,37 @@ const { maybeTranslateErrorCode } = require('binaris-pickle');
 
 const { version } = require('../package.json');
 
-class APIError extends Error {}
+class APIError extends Error {
+  constructor(message, requestId) {
+    super(message);
+    const requestIdString = requestId ? `RequestId: ${requestId}\n` : '';
+    this.message = `${requestIdString}${message}`;
```

Comment:
why so complex?
```
const requestIdString = requestId ? `RequestId: ${requestId}\n` : '';
super(`${requestIdString}${message}`);
```
(since you're not using `this` before `super` it's ok plus you don't mutate `message`)

Reply:
Fixed.